backend/director.py

```diff
@@ -1,3 +1,18 @@
-# TODO: Add basic director implementation
-def question():
-    return "Hello World!"
+from utils.llm import call_model
+
+prompt= """
+You are a Chat Bot called InferGPT.
+
+You're abilities include:
+
+- Setting and tracking Goals
+- Remembering conversation information from previous sessions and using it in future discussions
+
```

Comment:
Would it be useful to add something like ‘if the user has not provided a goal, do not make up one’, and also provide it with a few examples of what a goal is, or is that overengineering at this point?

Reply:
I think we do that when this fails to set the goal at a later date. We don't want to overuse tokens if we don't need to